l3wrapper/l3wrapper.py

```diff
@@ -232,6 +234,10 @@ def fit(self, X, y, column_names=None, save_human_readable=False, remove_files=T
         self.X_ = X
         self.y_ = y
 
+        tmp_dir = TemporaryDirectory()
```

Comment:
The point behind the **current_token_** is to help clients to keep track of the results generated by the last call to the **fit()** method. Now, every file that starts with *token_* is generated inside the temporary directory which makes it harder to access.

It might be useful to add the generated token to the name of the tmp_dir itself.
 ```suggestion
        token = secrets.token_hex(4)
        filestem = f"{token}"
        
        tmp_dir = TemporaryDirectory(prefix=token)
        old_dir = getcwd()
        chdir(tmp_dir.name)
```

Reply:
Ah, I understand now, yes, this seems like a good solution